api/auth/index.ts

```diff
@@ -107,10 +111,15 @@ export const getXuiNodeMiddleware = () => {
     useRoutes: true
   };
 
+  // Determine if session cookies should be used based on observed logins
+  function shouldUseSessionCookie(): boolean {
+    return idamLogins % 10 === 0; // Use session cookies for 1 in 10 sessions use a session cookie,
+  }
+
   const baseStoreOptions = {
     cookie: {
       httpOnly: true,
-      maxAge: 28800000,
+      maxAge: shouldUseSessionCookie() ? null : 28800000, // Adjust based on observed logins
```

Comment:
I don't like maxAge:null because it's not clear from the docs exactly what this means. 
Positive number - expire after than many secs
0 or negative expire immediately, effectively a 1 shot cookie I guess

Not set - expire when the browser is closed
But it's not completely clear that maxAge: null counts as not set.
Better to use the spread operator to add the attribute if shouldUseSessionCookie() is true